GVFS/GVFS.Common/PlaceholderListDatabase.cs

```diff
@@ -73,6 +69,15 @@ public static bool TryCreate(ITracer tracer, string dataFilePath, PhysicalFileSy
             return true;
         }
 
+        /// <summary>
+        /// The Count is "estimated" because it's simply (# adds - # deletes).  There is nothing to prevent
+        /// multiple adds or deletes of the same path from being double counted
+        /// </summary>
+        public int Count()

```

Comment:
nit: I would have made this be `public int Count => this.count;` so the external readers do not need to change to a method. PLUS: whenever I see `.Count()` I immediately think it is a reference to the LINQ method that should never be used.

Reply:
`Count()` is a part of the interface and I made it a method because I want users to think before using it because in the PlaceholderTable class it is making a query against the database and I don't want someone thinking it is just an in memory field that will be returned but is doing some work.

Reply:
> `Count()` is a part of the interface

I missed that. Thanks!